### ISSUE TITLE
Identify where on the website and how new WCAG guidance could be published

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#4684]
## What

Identify where on the website and how new WCAG guidance should be published.

A non-exhaustive list of questions: Does it make sense to publish something? Under which circumstances? Where does it belong? Should it be called out in any way? When should it mention WCAG and when should it be just general guidance?

This can only be done after we've decided which WCAG criterion we'll trial this on (#4682) and after we've checked to which components it applies (#4683).

## Why

We've written guidance for the new criteria in WCAG 2.2 (most of which we've removed again now). It makes sense to do it with the others, especially as most WCAG fails happen in other areas. We've also heard of some users that they are interested in such content.

## Who needs to work on this

Accessibility Specialist and Interaction Designer

## Who needs to review this

Anyone (Content Designer? DM? PM?)

## Done when

- [x] Decided it's a useful thing to do (or not)
- [x] Explored and documented where and how new guidance would fit
- [ ] ~Written example content to illustrate the above~

**Comment by selfthinker** [comment]
## General outcome

Ciandelle and me met today and went through the [spreadsheet](https://docs.google.com/spreadsheets/d/1y64TTi7MSYsbXdXRdzLhh3QioDXTwk8PLeGY7twAxNQ/edit?usp=sharing) to discuss if any of the suggested changes would be worth pursuing. We documented in the spreadsheet which one of them we found useful and which not.

Because we found 15 potential changes would be useful to add to the Design System website, we think adding more guidance in relation to WCAG is generally a good idea and something we should do.

## Which success criteria

Because I looked into what the most common fails are (as part of #4682), we think focussing on those makes the most sense. So, we don't need to do this exercise with all 56 success criteria (minus the 6 we already did as part of our WCAG 2.2 guidance updates). 
The Top 3 criteria alone cover 35.4% of all fails the monitoring team found in their detailed tests.
That's a good point to start.

The Top 12 (which cover about two thirds of fails) are:

- 1.4.3 Contrast (Minimum)
- 4.1.2 Name, Role, Value
- 1.3.1 Info and Relationships
- 2.4.7 Focus Visible
- 1.1.1 Non-text Content
- 4.1.1 Parsing - removed from WCAG 2.2
- 1.4.10 Reflow
- 2.1.1 Keyboard
- 1.4.11 Non-text Contrast
- 2.4.4 Link Purpose (In Context)
- 2.4.2 Page Titled
- 2.4.6 Headings and Labels
- 3.3.2 Labels or Instructions


## Other ideas

We thought that sometimes it might also be worth adding an accessibility section when there is more specific guidance on how to make / keep something accessible.

One idea was to show good and bad examples of things like [DfE's colour examples in their 'Accessibility and inclusive design manual'](https://accessibility.education.gov.uk/knowledge-hub/common-issues/colour).

Or when good guidance already exists elsewhere we could just add links to it. Like the DfE example above or the [WCAG Primer](https://alphagov.github.io/wcag-primer/) (which is being worked on to include GDS's [accessibility testing guidance](https://github.com/alphagov/wcag-primer/wiki)).

**Comment by selfthinker** [comment]
@Ciandelle, I have summarised our discussion above. Can you add if there is anything I have missed?
We can then call this task done.

There is an item in the "Done when" list that I don't think we'll have the time to do ("Written example content to illustrate the above"). I think it's fine to not do that and still call the task done.

**Comment by Ciandelle** [comment]
Absolutely perfect! 

Offline we also discussed the idea of aving a section of text within the accessibility section of the website to point users in the direction of blogs, videos and podcasts on accessibility that we’ve already got on the website.

Although we might be tempted to move them from the current location, they may cover more than one topic and therefore it could make it more difficult for users to find the information they are looking for. 

As a result, it would be better to write up something in the accessibility section of the Design System website to signpost users towards our resources. Proposed text could be something along the lines of:

> “There are a number of blogs, videos and podcasts about the topic of accessibility under the [community section of the website](https://design-system.service.gov.uk/community/blogs-talks-podcasts/). These resources offer practical advice on accessibility, as well as information about the work that has been done across government to make services more accessible and inclusive.”

**Comment by selfthinker** [comment]
Thanks Ciandelle. I think your suggestion fits better to the theme of adding new content the to accessibility page or section (#4677 and #4679).
I have made a small story for the latter (#4713) and have added your suggestion to that. Although it doesn't fit to that potential new page, it does fit to generally adding new content which could be covered by that.

I will close this now as the work is done.
